### ISSUE TITLE
java.lang.ExceptionInInitializeError when creating CNN

### ISSUE THREAD

**Issue by ChrisHayduk** [deeplearning4j/deeplearning4j#1395]
I and my team are currently trying to create a Convolutional Neural Network by following the online examples. However, we are getting an error when attempting to define the seed for the network. I have tried various different numbers for the seed, but they all create the same exception. The code and error are both listed below:

```java
public class Main {
    private static int iterations = 5;
    private static int num_people = 12;
    private static DataSetIterator iter;
    private static RecordReader recordReader;
    private static  String labeledPath = "/home/chris/Desktop/training/alex";

    public static void main(String[] args) throws Exception {
        loadData();
        System.out.println("Configuring model...");
        MultiLayerConfiguration.Builder builder = new NeuralNetConfiguration.Builder()
                .seed(1)
                .iterations(iterations)
                .regularization(true).l2(.0005) //Arbitrary
                .learningRate(0.01)
                .weightInit(WeightInit.XAVIER)
                .optimizationAlgo(OptimizationAlgorithm.STOCHASTIC_GRADIENT_DESCENT)
                .updater(Updater.NESTEROVS).momentum(0.9)
                .list(6)
                .layer(0, new ConvolutionLayer.Builder(5,5)
                    .nIn(11*200).nOut(20).activation("relu").padding(2,2).stride(1,1).dropOut(0.5).build()
                )
                .layer(1,new SubsamplingLayer.Builder(SubsamplingLayer.PoolingType.AVG)
                    .kernelSize(4,4).stride(1,1).build()
                )
                .layer(2, new ConvolutionLayer.Builder(5,5)
                    .nOut(20).activation("relu").stride(1,1).padding(2,2).dropOut(0.5).build()
                )
                .layer(3, new SubsamplingLayer.Builder(SubsamplingLayer.PoolingType.AVG)
                        .kernelSize(2,2).stride(1,1).build()
                )
                .layer(4, new DenseLayer.Builder().activation("relu").nOut(120).build())
                .layer(5, new OutputLayer.Builder().nOut(num_people).activation("softmax").build())
                .backprop(true).pretrain(false);
        new ConvolutionLayerSetup(builder,11,200,1);

        MultiLayerConfiguration conf = builder.build();
        MultiLayerNetwork model = new MultiLayerNetwork(conf);
        model.init();
        System.out.println("Training model....");
        while(iter.hasNext()){
            DataSet next = iter.next();
            System.out.println(Arrays.toString(next.getFeatureMatrix().shape()));
            System.out.println(Arrays.toString(next.getLabels().shape()));
            model.fit(next);
        }
        iter.reset();
       
        System.out.println("Testing model...");
        Evaluation eval = new Evaluation();
        while(iter.hasNext()){
            DataSet next = iter.next();
            INDArray predict2 = model.output(next.getFeatureMatrix());
            eval.eval(next.getLabels(), predict2);
        }

        System.out.println(eval.stats());


    }

    public static void loadData() throws Exception{
        List<String> labels = new ArrayList<>();

        for (File file : new File(labeledPath).listFiles()){
            labels.add(file.getName());
        }
        recordReader = new ImageRecordReader(11,200,true, labels);
        recordReader.initialize(new FileSplit(new File(labeledPath)));
        iter = new RecordReaderDataSetIterator(recordReader);


    }
}
```

```
Exception in thread "main" java.lang.ExceptionInInitializerError
	at org.deeplearning4j.nn.conf.NeuralNetConfiguration$Builder.seed(NeuralNetConfiguration.java:386)
	at edu.fordham.cis.wisdm.deep_biometrics.Main.main(Main.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
Caused by: java.lang.RuntimeException: org.nd4j.linalg.factory.Nd4jBackend$NoAvailableBackendException
	at org.nd4j.linalg.factory.Nd4j.initContext(Nd4j.java:4525)
	at org.nd4j.linalg.factory.Nd4j.<clinit>(Nd4j.java:141)
	... 7 more
Caused by: org.nd4j.linalg.factory.Nd4jBackend$NoAvailableBackendException
	at org.nd4j.linalg.factory.Nd4jBackend.load(Nd4jBackend.java:148)
	at org.nd4j.linalg.factory.Nd4j.initContext(Nd4j.java:4522)
	... 8 more
```

**Comment by agibsonccc** [comment]
Please ensure you have an nd4j backend defined.
